elife/docker-scripts/docker-wait-daemon

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+timeout="${1:-120}"
+socket="/var/run/docker.sock"
+
+echo "Waiting for socket $socket to be writable"
+# similar to elife/utils/wait_for_port but with Unix sockets (-U)
+timeout "$timeout" sh -c "while ! nc -q0 -w1 -U $socket </dev/null >/dev/null 2>&1; do sleep 1; done"
```

Comment:
I use `timeout` in strip-coverletter to prevent excessively long squashes. Good stuff. I would add a comment beneath your comment though explaining this soupy bash code. Something like

> poll docker socket once a second until timeout is reached (default 120 seconds)

and if `nc` has long forms for those parameters (`-q`, `-w`, `-U`), use those.

Reply:
Unfortunately, no long form